README.md

```diff
@@ -122,11 +144,19 @@ Most of your API calls are probably going to have more than one parameter.  You'
 In the definition object, you'll need to implement the toString() method in a manner that uniquely identifies the API call.
```

Comment:
one line above this change `You'll have to create a Definition Object.`  To `You'll have to create a LoaderParams Object.`